### DISCUSSION TITLE
The new styling isn't good

### DISCUSSION THREAD

**Discussion by JamesMcGhee** [GitbookIO/community#558] [2. Published content]
The new styling makes a mess of the cover image and isn't at all what you see in the editor
The hints are far harder to see ... overall I would say its a big step backward

Here is what it looks like live
![image](https://github.com/GitbookIO/community/assets/31243736/47f25e06-5310-4948-9b44-aac318f6c6da)

And here is what I see in the editor ... and what live used to look like
![image](https://github.com/GitbookIO/community/assets/31243736/b66b7a63-cdce-4dca-bad7-7285a721f2ac)

Aside from the banner being blown out you can also see the hints that should be highlighted are subdued ... why. If you want to go with this style at least make it an option that isn't held behind a paywall.

While we would love to pay you for your services you have sadly made it so we go team scale or not at all, as a solo dev I'm not going to pay for 5 slots for my little KB just to get some minor control over header coloring but your monitization is a separate issue

**Comment by MadeByPP** [comment]
I feel the same way. I mentioned this request in #554

**Comment by taszap** [comment]
Closing in favour of #554 as our team has provided a response in that thread 

(re-opened)

**Reply by JamesMcGhee** [reply]
Not the same, the note you linked is talking about links. 

I am nothing the whole design change. From the coloring of the hints, to the header scaling and yes the links as well. The whole style is a miss.

Beyond that I also noted that the editor uses the old style so writing and knowing what the result will look like is even harder now I have to publish review adjust review repeat... none of this is good or good well beyond links and what was addressed in that link you gave

**Reply by taszap** [reply]
Hi James, 

Thanks for sharing - I was referring to the below quote as it applied to the whole style and aligning the editor vs published experience. 

> We'll look at iterating on the readability of content, both in the editor and in published content, and align the 2 experiences."

We continue to review feedback and will make improvements (in the meantime you can use the "preview" function in a change request to see how the changes will appear on the published side).
